apps/user/src/components/form/DateBox/DateBox.tsx

```diff
@@ -39,7 +39,21 @@ const DateBox = () => {
                 }
                 value={date.year}
                 name="year"
-                data={['2006', '2007']}
+                data={[
+                    '2000',
+                    '2001',
+                    '2002',
+                    '2003',
+                    '2004',
+                    '2005',
+                    '2006',
+                    '2007',
+                    '2008',
+                    '2009',
+                    '2010',
+                    '2011',
+                    '2012',
+                ]}
```

Comment:
이런 규칙성있는 배열은 배열 메서드를 하면 좋을 것 같아요
```suggestion
                data={
                    Array(2012 - 2000 + 1).map((_, index) => (2000 + index).toString())
                }
```

처럼요